cpp/daal/include/algorithms/decision_forest/decision_forest_training_parameter.h

```diff
@@ -86,6 +86,16 @@ enum ResultToComputeId
     computeOutOfBagErrorPrediction       = 0x00000020ULL
 };
 
+/**
+ * <a name="DAAL-ENUM-ALGORITHMS__DECISION_FOREST__TRAINING__SPLITTER_MODE"></a>
+ * \brief Node splitting mode
+ */
+enum splitter_mode
```

Comment:
Use correct formatting for oneDAL library entities
```suggestion
enum SplitterMode
```